modules/ROOT/pages/media.adoc

```diff
@@ -20,7 +22,9 @@ tinymce.init({
 
 == Options
 
-These settings affect the execution of the `+media+` plugin. Namely the ability to disable parts of the media dialog box when inserting/editing media items. In addition, the user may disable the cross-site scripting sanitation filter for video/object elements here.
+These settings affect the execution of the `+media+` plugin, making it possible to disable parts of the media dialog box made available to an end-user inserting or editing media items.
```

Comment:
```suggestion
These options affect the execution of the `+media+` plugin, making it possible to disable parts of the media dialog made available to an end-user inserting or editing media items.
```
`settings` is the old name so just replaced it. Also `box` seemed redundant here and if anything confused me more 🤷